providers/common/capabilities.c

```diff
@@ -18,10 +18,11 @@
 #include "internal/tlsgroups.h"
 #include "prov/providercommon.h"
 #include "internal/e_os.h"
-#include "crypto/mlkem.h"
+#include "crypto/ml_kem.h"
 
 /* If neither ec or dh is available then we have no TLS-GROUP capabilities */
```

Comment:
Please add ml-kem to this comment so it better reflects reality.

Reply:
Sure.